### ISSUE TITLE
Error report

### ISSUE THREAD

**Issue by Lemony057** [krlvm/PowerTunnel-Android#111]
```
java.lang.RuntimeException: Error receiving broadcast Intent { act=io.github.krlvm.powertunnel.android.broadcast.FAILURE flg=0x10 (has extras) } in io.github.krlvm.powertunnel.android.activities.MainActivity$1@beeb236
	at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0$LoadedApk$ReceiverDispatcher$Args(LoadedApk.java:1676)
	at android.app.-$$Lambda$LoadedApk$ReceiverDispatcher$Args$_BumDX2UKsnxLVrE6UJsJZkotuA.run(Unknown Source:2)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:246)
	at android.app.ActivityThread.main(ActivityThread.java:8633)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
Caused by: android.view.WindowManager$BadTokenException: Unable to add window -- token android.os.BinderProxy@fa35a03 is not valid; is your activity running?
	at android.view.ViewRootImpl.setView(ViewRootImpl.java:1444)
	at android.view.WindowManagerGlobal.addView(WindowManagerGlobal.java:469)
	at android.view.WindowManagerImpl.addView(WindowManagerImpl.java:114)
	at android.app.Dialog.show(Dialog.java:505)
	at io.github.krlvm.powertunnel.android.activities.MainActivity$1.onReceive(MainActivity.java:142)
	at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0$LoadedApk$ReceiverDispatcher$Args(LoadedApk.java:1666)
	... 8 more
```

**Comment by farabox1** [comment]
What can I do ??

**Comment by farabox1** [comment]
Why ??

**Comment by krlvm** [comment]
I believe it has been fixed with [version 2.3.1](https://github.com/krlvm/PowerTunnel-Android/releases/tag/v2.3.1), please, provide the crash details otherwise.
